crates/routes/src/utils/scheduled_tasks.rs

```diff
@@ -359,6 +361,24 @@ async fn active_counts(pool: &mut DbPool<'_>) -> LemmyResult<()> {
     .execute(&mut conn)
     .await?;
 
+  let mut conn = get_conn(pool).await?;
+
+  let user_count: i64 = local_user::table
+    .inner_join(person::table.left_join(instance_actions::table))
```

Comment:
Filter this to only include `instance_actions` that are for the local instance.

Reply:
If they did something like blocking to other instances, then the join could return multiple rows for one user.

Reply:
Thats true, added a join to local site.